aidants_connect_web/templates/aidants_connect_web/home_page.html

```diff
@@ -4,41 +4,57 @@
 
 {% block content %}
 
-<section class="section">
-  <div class="container">
+{% block extracss %}
+  <link href="{% static 'css/home.css' %}" rel="stylesheet">
+{% endblock extracss %}
+
+<div class="hero">
+  <div class="hero__container">
     <div class="row">
-      <div class="hero__description">
+      <div class="hero__description text-center">
         <h1 class="brand">Bienvenue sur Aidants Connect</h1>
         <p class="lead-text">
           Le service qui permet aux aidants professionnels de réaliser des démarches administratives en ligne pour le compte d’une personne via une connexion sécurisée.
         </p>
-        <p>Ce service permet :
-          <ul>
-            <li>une protection du professionel déclarant comme de la personne aidée en cas d’erreur ;</li>
-            <li>un suivi des actions réalisées sur le compte de la personne.</li>
-          </ul>
-        </p>
-      </div>
-      <div class="column__image">
-        <img src="{% static 'images/aidantsconnect.png' %}" />
       </div>
     </div>
   </div>
-</section>
+</div>
+<div class="notification full-width">Vous êtes sur un site en construction, l’interface proposée sera amenée à changer grâce à vos retours !</div>
```

Comment:
Que penserais-tu de rajouter un lien vers la création d'une nouvelle issue GH ?
```suggestion
<div class="notification full-width">Vous êtes sur un site en construction, l’interface proposée sera amenée à changer grâce à vos [retours](https://github.com/betagouv/Aidants_Connect/issues/new) !</div>
```

Reply:
Excellente idée !